docs/man_pages/project/testing/publish-ios.md

```diff
@@ -0,0 +1,50 @@
+`<Password>` the Password for authentiation with iTunesConnect
+`<Mobile Provisioning Profile Identifier>` the identifier of the mobile provision which will be used for building. This can easily be acquired through the iPhone Configuration Utility
+`<Code Sign Identity>` the Code Sign Identity which will be used for building. It can be set to something generic like 'iPhone Distribution' and allow the build to automatically detect a Code Sign Identity.
+
+<% if(isHtml) { %>
+### Command Limitations
+
+* You can run `$ tns publish ios` only on OS X systems.
+
+### Related Commands
+
+Command | Description
+----------|----------
+[build android](build-android.html) | Builds the project for Android and produces an APK that you can manually deploy on device or in the native emulator.
+[build ios](build-ios.html) | Builds the project for iOS and produces an APP or IPA that you can manually deploy in the iOS Simulator or on device, respectively.
+[build](build.html) | Builds the project for the selected target platform and produces an application package that you can manually deploy on device or in the native emulator.
+[debug android](debug-android.html) | Debugs your project on a connected Android device or in a native emulator.
+[debug ios](debug-ios.html) | Debugs your project on a connected iOS device or in a native emulator.
+[debug](debug.html) | Debugs your project on a connected device or in a native emulator.
+[deploy](deploy.html) | Builds and deploys the project to a connected physical or virtual device.
+[emulate android](emulate-android.html) | Builds the specified project and runs it in a native Android emulator.
+[emulate ios](emulate-ios.html) | Builds the specified project and runs it in the native iOS Simulator.
+[emulate](emulate.html) | You must run the emulate command with a related command.
+[livesync](livesync.html) | Synchronizes the latest changes in your project to devices.
+[livesync ios](livesync-ios.html) | Synchronizes the latest changes in your project to iOS devices or the iOS Simulator.
+[livesync android](livesync-android.html) | Synchronizes the latest changes in your project to Android devices.
+[publish](publish.html) | Builds the project and publishes it to an Application Store.
+[run android](run-android.html) | Runs your project on a connected Android device or in a native Android emulator, if configured.
+[run ios](run-ios.html) | Runs your project on a connected iOS device or in the iOS Simulator, if configured.
+[run](run.html) | Runs your project on a connected device or in the native emulator for the selected platform.
+[test init](test-init.html) | Configures your project for unit testing with a selected framework.
+[test android](test-android.html) | Runs the tests in your project on Android devices or native emulators.
+[test ios](test-ios.html) | Runs the tests in your project on iOS devices or the iOS Simulator.
+<% } %>
+<% } else { %>
+	This command is not applicable to your current configuration.
```

Comment:
Usually we show the Synopsis and the short description and then say that the current command is not applicable to OS X systems.